### DISCUSSION TITLE
monster.ca: breakage

### DISCUSSION THREAD

**Discussion by prestr** [uBlockOrigin/uAssets#25444] [Q&A] [answered]
uBlock Origin blocks Monster.ca search results when a "Privacy" filter is enabled in Firefox. 
If used in Edge it works fine but I'd rather not use Edge. 

The Privacy filters I'm using are "EasyPrivacy" and "AdGuard Tracking Protection". Just disabling one or the other doesn't resolve the issue, it only resolves if both are disabled, so they much have something in common causing the issue in Firefox. 

Basically in Firefox, when I do a job search it says "Sorry, no jobs found for that search." but in Edge I get the results shown to me.

**Comment by Yuki2718** [comment]
Please report in https://github.com/uBlockOrigin/uAssets/issues with required steps. The discussion is not to bypass issue-reporting.

**Comment by gorhill** [comment]
@prestr Reporting filter issues by using the _chat_ icon in the popup panel is the best way to have filter issues looked at and fixed. Had you reported the issue through the _chat_ tool, it probably would have been fixed within hours if not minutes.

Anyway, I can reproduce the issue in Firefox, but not in Chromium. The problematic filter appears to be EasyPrivacy's `||datadome.co^$3p`, but for unknown reasons, only with Firefox.

---

More specifically, the site requires `https://js.datadome.co/tags.js`.

**Reply by stephenhawk8054** [reply]
`datadome` is for bot detection, so the result might vary between chromium and firefox I think

**Reply by Yuki2718** [reply]
@gorhill Are you logged in?

**Reply by gorhill** [reply]
No, I don't have an account on this site.

**Reply by Yuki2718** [reply]
Okay, I could reproduce only on Firefox as well.

**Reply by Yuki2718** [reply, marked as answer]
Fixed by https://github.com/easylist/easylist/commit/a42ff35c97bf1b5b21b5b7b92afd2cfe58e7399b

**Reply by prestr** [reply]
> @prestr Reporting filter issues by using the _chat_ icon in the popup panel is the best way to have filter issues looked at and fixed. Had you reported the issue through the _chat_ tool, it probably would have been fixed within hours if not minutes.
> 
> Anyway, I can reproduce the issue in Firefox, but not in Chromium. The problematic filter appears to be EasyPrivacy's `||datadome.co^$3p`, but for unknown reasons, only with Firefox.
> 
> More specifically, the site requires `https://js.datadome.co/tags.js`.

@gorhill Sorry about that, I wasn't sure what to do. Going forward I'll make sure to do that.

@Yuki2718 Thanks for the fix!
